docs/infra/api-markdown-documenter/api-documentation-layout.mjs

```diff
@@ -41,10 +44,20 @@ const supportDocsLinkSpan = new SpanNode([
  * @privateRemarks
  * If we later wish to differentiate between release tags of `@legacy` items, this function will need
  * to be updated.
+ *
+ * @param apiItem {import("@fluid-tools/api-markdown-documenter").ApiItem} - The API item for which the import notice is being created.
```

Comment:
Does the type specifier still work after the param name?

Reply:
Also on line 108.

Reply:
It seems to, yeah. Without it, intellisense complained about any method/property access on the variable. But you're right that this is not the right order per JSDoc spec. I'll do a quick follow-up to clean this up.